content/api/cli.md

```diff
@@ -3,15 +3,14 @@ title: CLI
 ---
 ## Overview
 
-`webpack` provides a command line interface(cli) to configure and interact with your build.
-This is mostly useful in case of early prototyping, profiling, writing npm scripts or personal customization of the build.
-For proper usage and easy distribution of this configuration, webpack can be configured with `webpack.config.js`.
-Any parameters sent to the CLI will map to a corresponding parameter in the config file.
+webpack provides a Command Line Interface(CLI) to configure and interact with your build.This is mostly useful in case of early prototyping, profiling, writing npm scripts or personal customization of the build.For proper usage and easy distribution of this configuration, webpack can be configured with `webpack.config.js`.Any parameters sent to the CLI will map to a corresponding parameter in the config file.
```

Comment:
Spacing.

* `Interface(CLI)` -> `Interface (CLI)`.
* `build.For` -> `build. For`

One at the end of the paragraph too.